package.json

```diff
@@ -157,7 +157,8 @@
     "patchedDependencies": {
       "qunit@2.19.4": "patches/qunit@2.19.4.patch",
       "testem@3.11.0": "patches/testem@3.11.0.patch",
-      "@ember/test-helpers@3.3.0": "patches/@ember__test-helpers@3.3.0.patch"
+      "@ember/test-helpers@3.3.0": "patches/@ember__test-helpers@3.3.0.patch",
```

Comment:
do we still need both?

Reply:
@runspired yes we need both, since alle patches will be added into v3 & v4 of test-helpers...

Reply:
the answer here is "we only need the 3.x patch because of the ember-try scenario for 3.28", I don't want to remove that scenario just yet so left it for now